tests/stress/stress_test.go

```diff
@@ -30,7 +23,7 @@ func NewStressTest(t Tester, dir string, vars map[string]string) *StressTest {
 		tftest.Dir(dir),
 		tftest.Vars(vars),
 		tftest.DryRun(*dry),
-		tftest.Log(log))
+		tftest.Log(t.(tftest.Logger)))
```

Comment:
this will cause a panic

Reply:
Will it? It seemed to be happy when I ran the stress tests. Why would this cause a panic?

Reply:
Only if `t` can't implement both `Tester` and `Logger` though, right? I didn't get any complaints when I ran a test locally, but that's probably because `testing.B` can be both a `Tester` and a `Logger`.

I understand, though, that the func sig doesn't actually enforce that behavior, so in the event that a user passes in a `Tester` that doesn't also implement `Logger`, Go would fall over. 👍 ✅